packages/editor/src/lib/components/default-components/DefaultCanvas.tsx

```diff
@@ -499,32 +499,39 @@ function DebugSvgCopy({ id }: { id: TLShapeId }) {
 		const unsubscribe = react('shape to svg', async () => {
 			const renderId = Math.random()
 			latest = renderId
-			const bb = editor.getShapePageBounds(id)
-			const el = await editor.getSvg([id], {
+			const result = await editor.getSvgString([id], {
 				padding: 0,
 				background: editor.getInstanceState().exportBackground,
 			})
-			if (el && bb && latest === renderId) {
-				el.style.setProperty('overflow', 'visible')
-				el.setAttribute('preserveAspectRatio', 'xMidYMin slice')
-				el.style.setProperty('transform', `translate(${bb.x}px, ${bb.y + bb.h + 12}px)`)
-				el.style.setProperty('border', '1px solid black')
-				setHtml(el?.outerHTML)
-			}
+
+			if (latest !== renderId || !result) return
+
+			const svgDataUrl = `data:image/svg+xml;utf8,${encodeURIComponent(result.svg)}`
+			setSrc(svgDataUrl)
 		})
 
 		return () => {
 			latest = null
 			unsubscribe()
 		}
 	}, [editor, id, isInRoot])
+	const bb = editor.getShapePageBounds(id)
 
-	if (!isInRoot) return null
+	if (!isInRoot || !src || !bb) return null
 
 	return (
```

Comment:
the debug mode seems to not work at all in this PR. it's currently broken in staging too, but only by the svg offsets being out of whack. Here the svgs don't seems to show up at all?

Reply:
huh, this was some weird css we had in .com so i wasn't seeing this issue on the examples app. fixed now!